### PR TITLE
amp-ad file size fix : DO NOT import from integration.js

### PR DESCRIPTION
**Pull request by zhouyx**

### CODE REVIEW

File: 3p/integration.js [hunk 1]
```diff
@@ -200,14 +200,6 @@ const defaultAllowedTypesInCustomFrame = [
   '_ping_',
 ];
 
-// List of ad networks that will manually call `window.context.renderStart` to
-// emit render-start event when ad actually starts rendering. Please add
-// yourself here if you'd like to do so (which we encourage).
-export const waitForRenderStart = [
```
Comment (cramforce) [inline 3p/integration.js]: Interesting: This file doesn't need it at all?

**Comment by cramforce** [comment]
LGTM

Thanks!

**Comment by erwinmombay** [comment]
@zhouyx can you sanity check by running `gulp clean && gulp build && gulp dist` then do `gulp size` and commit the new `size.txt` file?

File: 3p/integration.js (hunk 1 above)
Comment (zhouyx) [inline 3p/integration.js]: We used to use it in `integration.js`. But now we decide to always send `bootstrap-loaded` (used to be `render-start`) message, and let `amp-ad` decide whether to listen to it or not.

**Comment by zhouyx** [comment]
@erwinmombay the new size for amp-ad-0.1js min is 34.89kB.

**Comment by erwinmombay** [comment]
LGTM

**Comment by zhouyx** [comment]
shit! I didn't update doc! 😢

**Pull request merged**